### PR TITLE
common: disconnect a slave on-demand

### PR DESCRIPTION
**Pull request by naumvd95**

### CODE REVIEW

File: servers/partner-ci/common.yaml [hunk 1]
```diff
@@ -198,6 +198,13 @@
                   rm -rf *.rpm
                   sudo cp /var/log/libvirt/libvirtd.log ${{WORKSPACE}}/libvirtd_total.log
                   sudo chown jenkins:jenkins ${{WORKSPACE}}/libvirtd_total.log
+      - postbuildscript:
+          script-only-if-succeeded: False
+          script-only-if-failed: True
+          builders:
+              - shell: |
+                  [[ ${{DISCONNECT_ON_FAILURE}} ]] && \
+                  python -c "import jenkins; jenkins.Jenkins('http://172.16.177.5:8080', username='rkhozinov', password='7d0afaf10fe8866eed8c2f43cac83fbf').disable_node('${{NODE_NAME}}')"
```
Comment (rkhozinov) [inline servers/partner-ci/common.yaml]: `python -c "import jenkins; jenkins.Jenkins($JENKINS_URL, username='$JENKINS_USERNAME', password='$JENKINS_PASSWORD'`

File: servers/partner-ci/common.yaml [hunk 2]
```diff
@@ -210,12 +217,6 @@
                     property-file: reporter.properties
                     fail-on-missing: true
                     condition: FAILED_OR_BETTER
-      - conditional-publisher:
```
Comment (rkhozinov) [inline servers/partner-ci/common.yaml:213]: Didn't work with boolean-expression?

File: servers/partner-ci/9.0/manila.yaml [hunk 3]
```diff
@@ -242,7 +242,7 @@
             cd ${{WORKSPACE}}
             for refspec in ${{MULTI_REFS}}; do
               git fetch ${{GIT_URL}} "$refspec" && git cherry-pick FETCH_HEAD || git status
-              git log --pretty=format:'%h %ad | %s%d [%an]' --graph --date=short
+              git log  --graph --date=short --pretty=format:%h %ad | %s%d [%an]
```
Comment (rkhozinov) [inline servers/partner-ci/9.0/manila.yaml]: filter with `head`

File: servers/partner-ci/common.yaml (hunk 2 above)
Comment (naumvd95) [reply servers/partner-ci/common.yaml:213]: there is no shell-case in publishers

**Comment by rkhozinov** [comment]
closes #5

**Review by rkhozinov — APPROVED** [review]

**Comment by rkhozinov** [comment]
closes #17

**Comment by rkhozinov** [comment]
closes #5

**Pull request merged**